src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,80 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY_TEXT = "supply";
+    private static final String BUY_TEXT = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String line = readFromFile(fromFileName);
+        String report = createReport(line);
+        writeToFile(toFileName, report);
+    }
+
+    public String readFromFile(String fromFileName) {
+        File incomingFile = new File(fromFileName);
+        String readString;
+        StringBuilder readerBuilder = new StringBuilder();
+        try (BufferedReader reader = new BufferedReader(new FileReader(incomingFile))) {
+            String fileLine = reader.readLine();
+            while (fileLine != null) {
+                readerBuilder.append(fileLine).append(",");
+                fileLine = reader.readLine();
+            }
+            readString = readerBuilder.toString();
+            readerBuilder.setLength(0);
+            return readString;
+        } catch (IOException b) {
+            throw new RuntimeException("Cannot read from file" + b.getMessage());
+        }
+    }
+
+    public String createReport(String line) {
```

Comment:
lots of literals and static numbers that should be constants, e.g. `2` `1` ` ` `,` `result`